enhancements/installer/ipi-aws-china.md

```diff
@@ -0,0 +1,130 @@
+---
+title: ipi-install-aws-china
+authors:
+  - "@wanghaoran1988"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-02-14
+last-updated: 2020-02-14
+status: implementable
+---
+
+# IPI install on AWS China
+
+## Release Sign off Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+We have customers want to install OpenShift on AWS China, but currently
+OpenShift installer doesn't show up the AWS China regions `cn-north-1` and `cn-northwest-1`.
+These two regions are isolated from other global regions and RHCOS AMI pushed to global cannot
+be used in AWS China, [ARNs and api endpoint](https://docs.amazonaws.cn/en_us/aws/latest/userguide/endpoints-arns.html) are different.
+
+Similar to other global regions, we should support these two regions for customer who want deploy
+OpenShift on AWS China.
+
+## Motivation
+
+### Goals
+
+* OpenShift installer support IPI install on AWS China Regions.
+* CI job executing testings on AWS China regions.
+
+### Non-Goals
+
+* It's not a goal to detail how to request and setup a AWS account in AWS China.
+* It's not a goal to detail how to do UPI install.
+
+## Proposal
+
+In order to support install OpenShift on AWS China, we need:
+
+* Setup a public AWS China Account to host RHCOS AMIs.
+* Push RHCOS AMIs to AWS China account, and share them to public.
+* OpenShift installer support AWS China Regions.
+* All OCP components using AWS apis should use AWS China api endpoints.
+
+### User Stories
+
+#### Setup AWS China Account
+
+The AMIs in global regions are not useable in AWS China regions, we need setup an AWS China Account to host our RHCOS AMIs, so that installer can use them to setup the cluster.
+
+#### Push RHCOS AMIs to AWS China account, and share them to public.
+
+Currently, we have CI jobs push the AMIs to public regions, after the AWS China account setup is ready, we should make our CI job start push our AMIs to AWS China regions, and share them to all accounts in AWS China regions.
+
+#### OpenShift installer support AWS China regions
+
+The OpenShift installer should be able to use the AMIs that pushed to AWS China regions to provision clusters, and use the correct api endpoints and ARNs, Notable difference for AWS China:
+
+* AWS resources ARNs in China regions are prefixed with "arn:arn-cn"
+* Ec2 service endpoint is "ec2.amazonaws.com.cn"
```

Comment:
AWS SDK already has endpoint information for this region, so i think we should probably focus on 
1) make sure you are using SDK defaults
2) define how the SDK should be configured so that (1) is achievable.